translations/es/translation.json

```diff
@@ -13,7 +13,28 @@
       "table": "",
       "xml": ""
     },
+    "friendlyParamError": {
```

Comment:
Maybe open an issue for a request for a spanish translation of these once this gets merged?

Reply:
I was thinking of a general issue for all languages. But in #4233 @outofambit and @limzykenneth talked about how including translations as part of the main library file may increase the size quite a lot, and solving this by perhaps serving the translation files sepearately. Do we want this to be addressed first before people start contributing the translations?

Reply:
Right. We can address that first. I just want to make sure that an issue gets opened so we don't forget that the more general concern needs addressing.